File: python/2.py
import json
import boto3
import operator

def lambda_handler(event, context):
   

    client = boto3.client('s3')
    s3 = boto3.client('s3')
    data = s3.get_object(Bucket='ec-lamba-test', Key='fadetoblack.txt')
    contents = data['Body'].read().decode('utf-8') 
    
    words=contents.split()
    word_count = dict()
 
    for word in words:
        if word.lower() in word_count:
            word_count[word.lower()] += 1
        else:
            word_count[word.lower()]=1
    
    sorted_words = sorted(word_count.items(), key=lambda x: x[1], reverse=True)


    return sorted_words
    
    #return contents
    #print(contents)

File: node/2.js
const AWS = require('aws-sdk');
const params = {Bucket: 'ec-lamba-test', Key: 'fadetoblack.txt'};
var returndata = "";

exports.handler = (event, context, callback) => {
    var s3 = new AWS.S3({apiVersion: '2006-03-01'});
    
    s3.getObject(params, function(err, data) {
    if (err) console.log(err, err.stack); // an error occurred
     else {
        returndata = Buffer.from(data.Body).toString('utf8').replace(/\n/g, ' ').toLowerCase().split(' ');
         
        var word_count = {}
         
        returndata.forEach(function(entry) {
            if(word_count[entry]){word_count[entry] ++; }
            else{word_count[entry] = 1 }
        });
        
        
    
        console.log(word_count);
        const response = {
            statusCode: 200,
            body: word_count
        };
    callback(null, response);
         }// successful response
    });
    
};
